test/config/100-strimzi-cluster-operator-0.16.2.yaml

```diff
@@ -7,12 +7,12 @@ metadata:
 spec:
   group: kafka.strimzi.io
   versions:
```

Comment:
I have a script ;-) that downloads the latest ....   and not sure if @chizhg would be intersted in that kinda stuff

Reply:
Yeah, I was mainly just experimenting with why the tests were failing. When I tried to run them on my own cluster, I could get them to pass every now and then. Looks like progress is happening. I'm going to try again with parallelism since I'm seeing some wonky things with kafka channel with multiple subscriptions...